projects/js-packages/publicize-components/src/components/manage-connections-modal/confirmation-form/index.tsx

```diff
@@ -103,7 +103,7 @@ export function ConfirmationForm( {
 		}
 
 		if (
-			service.multiple_external_user_ID_support &&
+			service.supports.additional_users_only &&
```

Comment:
This should have been 

```suggestion
			service.supports.additional_users &&
```